### ISSUE TITLE
Missing line in file developer-guide.md

### ISSUE THREAD

**Issue by sanjusss** [dotnet/spark#99]
https://github.com/dotnet/spark/blob/master/docs/developer-guide.md#debugging-net-application  
debug command：
```
spark-submit \
  --class org.apache.spark.deploy.DotnetRunner \
  --master local \
  debug
```
The correct way is：
```
spark-submit \
  --class org.apache.spark.deploy.DotnetRunner \
  --master local \
  <path-to-microsoft-spark-jar> \
  debug
```

**Comment by rapoth** [comment]
Thanks for reporting. CC: @imback82

**Comment by imback82** [comment]
Addressed by #100
